motion_tracker/model/eval_model.py

```diff
@@ -0,0 +1,41 @@
+import os
+import pandas as pd
+import seaborn as sns
+from matplotlib import pyplot as plt
+from motion_tracker.model.model import make_model
+from motion_tracker.utils.image_generator import AlovGenerator
+
+if __name__ == "__main__":
+    img_edge_size = 224
+    backend_id = 'th'
+    error_analysis_dir = 'work/error_analysis/'
```

Comment:
I think to have this line up with the below we should have './work/error_analysis'. Doesn't really matter, though.